flow/connectors/clickhouse/normalize.go

```diff
@@ -278,170 +280,202 @@ func (c *ClickHouseConnector) NormalizeRecords(
 		return nil, err
 	}
 
-	rawTbl := c.getRawTableName(req.FlowJobName)
+	parallelNormalize, err := peerdbenv.PeerDBClickHouseParallelNormalize(ctx, req.Env)
+	if err != nil {
+		return nil, err
+	}
 
-	// model the raw table data as inserts.
-	for _, tbl := range destinationTableNames {
-		// SELECT projection FROM raw_table WHERE _peerdb_batch_id > normalize_batch_id AND _peerdb_batch_id <= sync_batch_id
-		selectQuery := strings.Builder{}
-		selectQuery.WriteString("SELECT ")
+	var destinationTableNameChunks iter.Seq[[]string]
```

Comment:
can we log something to indicate we are doing parallel normalize along with the number